### ISSUE TITLE
Type information can not be found

### ISSUE THREAD

**Issue by mgrandrath** [nerdyman/react-image-turntable#32]
When building the package the declaration file `dist/index.d.mts` is created. However `package.json` states `"typings": "dist/index.d.ts"` ([code](https://github.com/nerdyman/react-image-turntable/blob/9b9b8ab68e9f13bd1521e52d66f303131d824471/package.json#L25)). Because of this mismatch (`.mts` vs `.ts`) TypeScript fails to find the type information and gives the error `Cannot find module 'react-image-turntable' or its corresponding type declarations`.

**Comment by nerdyman** [comment]
Whoops, thanks for raising this @mgrandrath. I'll publish a fix release to resolve this.

**Comment by densekernel** [comment]
This issue also occurs with `react-scripts test`:

```
 FAIL  src/App.test.js
  ● Test suite failed to run

    Cannot find module 'react-image-turntable' from 'src/GridView.js'
```

**Comment by nerdyman** [comment]
<img width="422" alt="image" src="https://github.com/nerdyman/react-image-turntable/assets/4908432/93cba92e-5c4e-465b-810d-f3dcba8f3467">

This has been resolved in `3.1.2`

https://arethetypeswrong.github.io/?p=react-image-turntable%403.1.2
